mlflow/catboost.py

```diff
@@ -39,17 +39,18 @@
 _MODEL_BINARY_FILE_NAME = "model.cb"
 
 
+def _get_default_pip_requirements():
+    import catboost as cb
+
+    return ["catboost=={}".format(cb.__version__)]
+
+
 def get_default_conda_env():
     """
     :return: The default Conda environment for MLflow Models produced by calls to
              :func:`save_model()` and :func:`log_model()`.
     """
-    import catboost as cb
-
-    return _mlflow_conda_env(
-        # CatBoost is not yet available via the default conda channels, so we install it via pip
-        additional_pip_deps=["catboost=={}".format(cb.__version__)]
-    )
+    return _mlflow_conda_env(additional_pip_deps=_get_default_pip_requirements())
```

Comment:
What this PR does is extract the code to prepare pip requirements from `get_default_conda_env` and redefine it as `_get_default_pip_requirements`. That's it.